ptrack-gentest/readme.txt

```diff
@@ -0,0 +1,2 @@
+requires octave to run (install with apt-get install octave in ubuntu)
```

Comment:
Better re-write the file as Markdown.
Use standard name convention as "README.md"

Reply:
corrected